spec/index.html

```diff
@@ -757,6 +757,12 @@ <h2>Literals</h2>
       The third element, the language tag, is treated identically as in a <a>language-tagged string</a>,
       and the fourth element, <a>base direction</a>, MUST be either `ltr` or `rtl`, which MUST be in lower case.</p>
 
+    <p>The meanings of the <a>base direction</a> values are:</p>
+    <ul>
+      <li>`ltr`: indicates that the initial text direction is set to left-to-right text.</li>
+      <li>`rtl`: indicates that the initial text direction is set to right-to-left text.</li>
```

Comment:
"... left-to-right text" is a bit redundant. Perhaps:

```suggestion
      <li>`ltr`: indicates that the initial text direction is set to left-to-right.</li>
      <li>`rtl`: indicates that the initial text direction is set to right-to-left.</li>
```